File: ympa_fsharp_giraffe_server/src/model/GenerateShipmentListDocumentReportRequest.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic

module GenerateShipmentListDocumentReportRequest =

  //#region GenerateShipmentListDocumentReportRequest


  type GenerateShipmentListDocumentReportRequest = {
    CampaignId : int64;
    ShipmentId : int64;
    OrderIds : int64[];
  }
  //#endregion

File: src/gatling/scala/org/openapitools/client/model/GenerateShipmentListDocumentReportRequest.scala

package org.openapitools.client.model


case class GenerateShipmentListDocumentReportRequest (
    /* Идентификатор кампании. */
    _campaignId: Long,
    /* Идентификатор отгрузки. */
    _shipmentId: Option[Long],
    /* Фильтр по идентификаторам заказа в отгрузке. */
    _orderIds: Option[List[Long]]
)
object GenerateShipmentListDocumentReportRequest {
    def toStringBody(var_campaignId: Object, var_shipmentId: Object, var_orderIds: Object) =
        s"""
        | {
        | "campaignId":$var_campaignId,"shipmentId":$var_shipmentId,"orderIds":$var_orderIds
        | }
        """.stripMargin
}
